framework/src/index.js

```diff
@@ -1,4 +1,5 @@
 const Application = require('./controller/application');
+const GenesisBlockDevnet = require('./modules/chain/defaults/genesis_block_devnet');
```

Comment:
As of now we have no reference or usage of concept `network` int he framework. Network is just the part of the Lisk Core implementation. So I suggest we simply rename it to `sampleGenesisBlock` and don't opt Network naming convention in the framework (at least in current state).

Reply:
I moved `genesis_block_devnet` to `framework/samples` directory I exported all the samples from there. 

It is a Genesis Block for development purposes, so `devnet` genesis block is correct to me.